lib/metadata/webkit.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+var bugsWebkit = require('../bugs-webkit');
+
+exports.related = function(title) {
+    return title.indexOf("WebKit export") > -1;
+};
+
+exports.verified = function(webkitIssue, wptPullRequestNumber) {
+        return bugsWebkit.get("/rest/bug/:id", { id: webkitIssue }).then(function (body) {
+            if (body && body.bugs && body.bugs.length > 0 && body.bugs[0].see_also) {
+                return body.bugs[0].see_also.some(function(url) {
+                    return url == "https://github.com/web-platform-tests/wpt/pull/" + wptPullRequestNumber;
+                });
+            }
+            return false;
+        });
+};
+
+exports.flags = function(issue) {
+        var flags = {
+            reviewed: false,
+            inCommit: false
+        };
+        return bugsWebkit.get("/rest/bug/:id/history", { id: issue }).then(function (body) {
+            body.bugs[0].history.forEach(function(history_item) {
+                // console.log("When:" + history_item.when);
+                history_item.changes.forEach(function(change) {
+                    // console.log("Change - added:" + change.added);
+                    // console.log("Change - removed:" + change.removed);
+                    // console.log("Change - field_name:" + change.field_name);
+                    if (change.field_name == "flagtypes.name") {
+                        if (change.removed.includes("review+")) {
+                            flags.reviewed = false;
+                        }
+                        if (change.removed.includes("commit-queue+")) {
+                            flags.reviewed = false;
+                            flags.inCommit = false;
+                        }
+                        if (change.added.includes("review+")) {
+                            flags.reviewed = true;
+                        }
+                        if (change.added.includes("commit-queue+")) {
+                            flags.reviewed = true;
+                            flags.inCommit = true;
+                        }
+                    }
+                    if (change.field_name == "resolution") {
+                        if (change.added.includes("FIXED")) {
```

Comment:
Is it possible for a WebKit bug to be closed FIXED without the change actually being committed to the codebase?

Reply:
Yes. It is possible set it as `Fixed` manually. 

I will change the condition as follows:

``` diff
@@ -45,7 +44,7 @@ exports.flags = function(issue) {
                         }
                     }
                     if (change.field_name == "resolution") {
-                        if (change.added.includes("FIXED")) {
+                        if (change.added.includes("FIXED") && change.who == "commit-queue@webkit.org" ) {
                             flags.reviewed = true;
                             flags.inCommit = true;
                         }
```

to ensure that the Fixed was done by the commit-queue.